dizoo/smac/config/smac_3s5z_madqn_config.py

```diff
@@ -0,0 +1,96 @@
+from ding.entry import serial_pipeline
+from easydict import EasyDict
+
+agent_num = 8
+collector_env_num = 4
+evaluator_env_num = 8
+
+main_config = dict(
+    #exp_name='smac_3s5z_siql_global_boost_dim256',
+    exp_name='3s5z_global_eval_seed1',
+    env=dict(
+        map_name='3s5z',
+        difficulty=7,
+        reward_only_positive=True,
+        mirror_opponent=False,
+        agent_num=agent_num,
+        collector_env_num=collector_env_num,
+        evaluator_env_num=evaluator_env_num,
+        stop_value=1.999,
+        n_evaluator_episode=32,
+        special_global_state=True,
+        manager=dict(
+            shared_memory=False,
+        ),
+    ),
+    policy=dict(
+        model=dict(
+            agent_num=agent_num,
+            obs_shape=150,
+            global_obs_shape=295,
+            global_boost=True,
+            action_shape=14,
+            hidden_size_list=[256, 256],
+            mixer=False,
+            lstm_type='gru',
+            dueling=False,
+        ),
+        learn=dict(
+            multi_gpu=False,
+            update_per_collect=20,
+            batch_size=64,
+            learning_rate=0.0005,
+            clip_value=5,
+            double_q=False,
```

Comment:
remove `double_q` and `iql` here, because there are the same as that in default config